### PR TITLE
ADEN-6715 | Add new sizes to MOBILE_BOTTOM_LEADERBOARD + filter sizes for UAP

### PR DESCRIPTION
**Pull request by kvas-damian**
https://wikia-inc.atlassian.net/browse/ADEN-6715

### CODE REVIEW

File: extensions/wikia/AdEngine/js/provider/directGptMobile.js [hunk 1]
```diff
@@ -15,7 +15,7 @@ define('ext.wikia.adEngine.provider.directGptMobile', [
 			INVISIBLE_HIGH_IMPACT:      {size: '1x1'},
 			INVISIBLE_HIGH_IMPACT_2:    {loc: 'hivi'},
 			MOBILE_TOP_LEADERBOARD:     {size: '300x50,320x50,320x100,320x480,2x2', loc: 'top'},
-			MOBILE_BOTTOM_LEADERBOARD:  {size: '320x480,2x2', loc: 'footer'},
+			MOBILE_BOTTOM_LEADERBOARD:  {size: '320x50,300x250,300x50,2x2', loc: 'footer'},
```
Comment (fraszczakszymon) [inline extensions/wikia/AdEngine/js/provider/directGptMobile.js:18]: Should we add BOTTOM_LEADERBOARD at this point?

Comment (fraszczakszymon) [reply extensions/wikia/AdEngine/js/provider/directGptMobile.js:18]: If yes... we need to handle it separately in slotContext based on skin

Comment (kvas-damian) [reply extensions/wikia/AdEngine/js/provider/directGptMobile.js:18]: Added

File: extensions/wikia/AdEngine/js/provider/gpt/adSizeFilter.js [hunk 2]
```diff
@@ -35,32 +34,39 @@ define('ext.wikia.adEngine.provider.gpt.adSizeFilter', [
...
 			adContext.targeting.hasFeaturedVideo
 		) {
-			slotSizes.forEach(function(size, index, array) {
-				var str = size.toString();
-
-				if (str === '2,2' || str === '3,3') {
-					array.splice(index, 1);
-				}
-			});
+			slotSizes = removeUAPFromSlotSizes(slotSizes);
```
Comment (v3rth) [inline extensions/wikia/AdEngine/js/provider/gpt/adSizeFilter.js:37]: <3

**Review by v3rth — APPROVED** [review]

**Pull request merged**